library/src-bootstrapped/scala/reflect/Enum.scala

```diff
@@ -0,0 +1,10 @@
+package scala.reflect
+
+/** A base trait of all enum classes */
+super trait Enum extends Any with Product, Serializable:
```

Comment:
added `Any` super class so it will be eliminated by drop super traits when the common superclass is AnyRef